### PR TITLE
11 insertions

### PR DESCRIPTION
**Pull request by littl3birdy**
- [ ] Top-level README “Table of Contents” is updated
 - [ x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ x] README for this challenge is complete
     - [x ] Summary, Description, Approach & Efficiency, Solution
     - [ x] Link to code
     - [ x] Picture of whiteboard

### CODE REVIEW

File: code401challenges/src/main/java/code401challenges/LinkedList.java [hunk 1]
```diff
@@ -49,5 +49,30 @@ public String toString(){
         }
         return linkedListOutput.toString();
     }
+    public void append(int value){
```
Comment (SharinaS) [inline code401challenges/src/main/java/code401challenges/LinkedList.java]: You may need to consider what happens if the head == null, as in, there's not yet a linked list. This also means, you may need to declare a new node above your logic statements.... 
```
if (head == null) {
   head = newNode
}
```
